.npmrc

```
@@ -0,0 +1 @@
+ignore-scripts=true
```

Having the registry in here is probably required.

Well maybe not since you're not installing any packages from here.  You're publishing them.  It'll be the consumer of your package that will need to put the registry in there.

I recommend adding:
```
engine-strict=true
omit-lockfile-registry-resolved=false
```

The first will hard fail instead of warn if an engine is used that doesn't match the semvers.

The second is the try to help make sure the lock file include the "resolved" and "integrity" fields that cachito/hermito have required in the past to ensure proper SBOM.